mlxtend/frequent_patterns/apriori.py

```diff
@@ -122,50 +141,80 @@ def apriori(df, min_support=0.5, use_colnames=False, max_len=None, verbose=0):
                              '`df.columns = [str(i) for i in df.columns`].')
 
         X = df.to_coo().tocsc()
-        support = np.array(np.sum(X, axis=0) / float(X.shape[0])).reshape(-1)
+        support = _support(X, X.shape[0], is_sparse)
     else:
         X = df.values
-        support = (np.sum(X, axis=0) / float(X.shape[0]))
+        support = _support(X, X.shape[0], is_sparse)
```

Comment:
Here, 

```python
        X = df.to_coo().tocsc()
         support = np.array(np.sum(X, axis=0) / float(X.shape[0])).reshape(-1)		         support = _support(X, X.shape[0], is_sparse)
     else:		     else:
         X = df.values		         X = df.values
         support = (np.sum(X, axis=0) / float(X.shape[0]))		         support = _support(X, X.shape[0], is_sparse)

``` 
the line

```python
support = _support(X, X.shape[0], is_sparse)
```

could be put after the if-else clause right? It looks identical and would be duplicated otherwise